data/Language/Verilog.yaml

```diff
@@ -0,0 +1,165 @@
+﻿identifier: Vlog
+full_name: Verilog
+wikidata: Q827773
+grammar_file: https://github.com/antlr/grammars-v4/blob/master/verilog/Verilog2001.g4
+creation_date: 2001
+aliases:
+  - Verilog 2001
+  - Verilog-2001
+  - IEEE 1364
+extensions:
+  - v
+  - vh
+line_continuation:
+  - '\'
+  - '|'
+  - '&'
+delimiters:
+  # comment delimiters
+  - double_slash
+  - multiline_slash_star
+  # string delimiters:
+  - double_quote_slash_escape
+  - curly_braces
+  - round_braces
+  - square_braces
+keywords:
+  - always
+  - and
+  - assign
+  - automatic
+  - begin
+  - buf
+  - bufif0
+  - bufif1
+  - case
+  - casex
+  - casez
+  - cell
+  - cmos
+  - config
+  - deassign
+  - default
+  - defparam
+  - design
+  - disable
+  - edge
+  - else
+  - end
+  - endcase
+  - endconfig
+  - endfunction
+  - endgenerate
+  - endmodule
+  - endprimitive
+  - endspecify
+  - endtable
+  - endtask
+  - event
+  - for
+  - force
+  - forever
+  - fork
+  - function
+  - generate
+  - genvar
+  - highz0
+  - highz1
+  - if
+  - ifnone
+  - initial
+  - instance
+  - inout
+  - input
+  - integer
+  - join
+  - large
+  - liblist
+  - localparam
+  - macromodule
+  - medium
+  - module
+  - nand
+  - negedge
+  - nmos
+  - nor
+  - not
+  - noshowcancelled
+  - notif0
+  - notif1
+  - or
+  - output
+  - parameter
+  - pmos
+  - posedge
+  - primitive
+  - pull0
+  - pull1
+  - pulldown
+  - pullup
+  - pulsestyle_onevent
+  - pulsestyle_ondetect
+  - rcmos
+  - real
+  - realtime
+  - reg
+  - release
+  - repeat
+  - rnmos
+  - rpmos
+  - rtran
+  - rtranif0
+  - rtranif1
+  - scalared
+  - signed
+  - showcancelled
+  - small
+  - specify
+  - specparam
+  - strength
+  - strong0
+  - strong1
+  - supply0
+  - supply1
+  - table
+  - task
+  - time
+  - tran
+  - tranif0
+  - tranif1
+  - tri
+  - tri0
+  - tri1
+  - triand
+  - trior
+  - trireg
+  - unsigned
+  - use
+  - vectored
+  - wait
+  - wand
+  - weak0
+  - weak1
+  - while
+  - wire
+  - wor
+  - xnor
+  - xor
+datatypes:
```

Comment:
The datatypes of verilog are ``` shortint, int, longint, byte, bit, logic, reg, integer, real, shortreal, realtime and time ``` remove unwanted ones and checkout the above listed ones.